### ISSUE TITLE
Superglobal array assignment

### ISSUE THREAD

**Issue by GaryJones** [WordPress-Coding-Standards/WordPress-Coding-Standards#734]
```php
$this->postData = $_POST;
```

Should the above trigger [this](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/51eb33b15f3aad5d74e1c816b5d8edc42163de25/WordPress/Sniffs/VIP/SuperGlobalInputUsageSniff.php#L60) error?

The [unit test file](https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/51eb33b15f3aad5d74e1c816b5d8edc42163de25/WordPress/Tests/VIP/SuperGlobalInputUsageUnitTest.inc) doesn't cover this - only accessing keys in the array, instead of the whole array itself.

**Comment by schlessera** [comment]
Although I was the one saying this was most likely a false positive (as it is a simple assignment, the contents are not accessed), I do think that it might not be possible to do a better check.

If this would not trigger the error, the error would have to be triggered on accessing the contents of `$this->postData`, which is not easily possible to correctly detect.

So, while I do think the above statement is not directly an issue, it might be the only chance of pointing out a potential issue before it gets obscured.

**Comment by GaryJones** [comment]
Even if that's the case, I think the unit tests should include such an instance to confirm the error would be triggered.

**Comment by jrfnl** [comment]
Closing as the sniff this issue applies to has been deprecated in WPCS 1.0.0 and removed in WPCS 2.0.0.
